### PR TITLE
fix(ReportsPartialsCampaigns): update content

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
image

image

### CODE REVIEW

File: src/components/Reports/ReportsPartialsCampaigns/ReportsPartialsCampaigns.js [hunk 1]
```diff
@@ -66,7 +66,12 @@ const ReportsPartialsCampaigns = ({ location, dependencies: { dopplerApiClient }
               <div className="dp-rowflex">
                 <div className="col-sm-12 col-md-12 col-lg-12">
                   <h3>
-                    <FormattedMessage id="reports_partials_campaigns.header_title" />
+                    <FormattedMessage
+                      id={
+                        'reports_partials_campaigns.header_title_' +
+                        state.campaignSummaryResults.campaignStatus
```
Comment (andresmoschini) [inline src/components/Reports/ReportsPartialsCampaigns/ReportsPartialsCampaigns.js:72]: I am not sure and it is arguable. 

Would template strings be cleaner than concatenation?

**Comment by andresmoschini** [comment]
Since you are translating this page, what do you think about also apply localized number formats for the numbers?

image

---

BTW, why the values are inside an `<h2>` element? it is clearly not a _header_. CC: @GusBaamonde

File: src/i18n/en.js [hunk 2]
```diff
@@ -204,25 +204,28 @@ by Doppler and how many don't. By tracking the user's journey you'll be able to
     visits_without_email: `Users without email`,
   },
   reports_partials_campaigns: {
-    campaign_name: `Campaign: `,
-    campaign_state: `Campaign State: `,
+    campaign_name: `Campaign name: `,
+    campaign_state: `Campaign status`,
```
Comment (lucianaDilizio) [inline src/i18n/en.js:208]: Maybe Name and Status should have the first letter in uppercase. In spanish is: Nombre de la  Campaña

File: src/i18n/es.js [hunk 3]
```diff
@@ -205,25 +205,28 @@ los usuarios, detecta puntos de fuga y oportunidades de mejora! Si necesitas ayu
...
+    campaign_name: `Nombre de la  Campaña: `,
+    campaign_state: `Estado de la Campaña `,
     campaign_subject: `Asunto: `,
     campaign_summary: `Resumen de la Campaña`,
     delivery_rate: `Tasa de entrega`,
     emails_delivered: `Emails Entregados:`,
-    hard_and_soft: `Rebotes hard y soft`,
-    header_title: `Reportes parcial de Campañas`,
+    hard_and_soft: `Rebotes Hard y Soft`,
+    header_title_sent: `Reporte Campaña`,
```
Comment (cbernat) [inline src/i18n/es.js:215]: Here perhaps Reporte de Campaña?

File: src/i18n/en.js (hunk 2 above)
Comment (cbernat) [reply src/i18n/en.js:208]: We only use capital letters for the start of the sentence and for Doppler particular objects like Campaigns, Lists, etc

**Comment by cbernat** [comment]
I see this a bit distoted, for sent campaigns, can you double check? https://cdn.fromdoppler.com/doppler-webapp/build2298/#/reports/partials-campaigns?campaignId=10937666

image

File: src/i18n/es.js [hunk 4]
```diff
@@ -205,25 +205,28 @@ los usuarios, detecta puntos de fuga y oportunidades de mejora! Si necesitas ayu
...
     last_click_date: `Último Click:`,
     last_open_date: `Última Apertura:`,
     not_open: `No Abiertos`,
     opened: `Abiertos`,
-    page_description: `Reportes parcial de Campañas`,
-    page_title: `Doppler | Reportes parcial de Campañas`,
+    page_description: `Reporte parcial`,
+    page_title: `Doppler | Reporte parcial`,
+    sent: `Enviada`,
+    shipping: `Enviando`,
```
Comment (cbernat) [inline src/i18n/es.js:224]: Here it should be `shipped`

File: src/i18n/es.js [hunk 5]
```diff
@@ -205,25 +205,28 @@ los usuarios, detecta puntos de fuga y oportunidades de mejora! Si necesitas ayu
...
+    campaign_state: `Estado de la Campaña `,
     campaign_subject: `Asunto: `,
     campaign_summary: `Resumen de la Campaña`,
     delivery_rate: `Tasa de entrega`,
     emails_delivered: `Emails Entregados:`,
-    hard_and_soft: `Rebotes hard y soft`,
-    header_title: `Reportes parcial de Campañas`,
+    hard_and_soft: `Rebotes Hard y Soft`,
+    header_title_sent: `Reporte Campaña`,
+    header_title_shipping: `Reporte Parcial`,
```
Comment (cbernat) [inline src/i18n/es.js:216]: Same here, state is `shipped`

File: src/i18n/es.js (hunk 4 above)
Comment (andresmoschini) [reply src/i18n/es.js:224]: 😕 

`sent : Enviada` or `shipped: Enviada`?
 
`shipping: Enviando` or `shipped: Enviando`?

Comment (cbernat) [reply src/i18n/es.js:224]: Yes I've noticed that I think that `shipped` is the state for sent, and shipping is the state for `in progress`

File: src/i18n/en.js (hunk 2 above)
Comment (fcoronelmakingsense) [reply src/i18n/en.js:208]: Talked with Mariano and he says its ok this works are in lowercase, Campaign Reports and Emails its ok in uppercase

File: src/components/Reports/ReportsPartialsCampaigns/ReportsPartialsCampaigns.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/components/Reports/ReportsPartialsCampaigns/ReportsPartialsCampaigns.js]: changed

File: src/i18n/es.js (hunk 3 above)
Comment (fcoronelmakingsense) [reply src/i18n/es.js]: changed

File: src/i18n/es.js (hunk 4 above)
Comment (fcoronelmakingsense) [reply src/i18n/es.js:224]: changed

File: src/i18n/es.js (hunk 5 above)
Comment (fcoronelmakingsense) [reply src/i18n/es.js:216]: changed

**Comment by GusBaamonde** [comment]
> BTW, why the values are inside an `<h2>` element? it is clearly not a _header_. CC: @GusBaamonde

If I was wrong, it would have to be a class for the size

File: src/components/Reports/ReportsPartialsCampaigns/ReportsPartialsCampaigns.js [hunk 6]
```diff
@@ -189,7 +197,9 @@ const ReportsPartialsCampaigns = ({ location, dependencies: { dopplerApiClient }
                             </li>
                             <li>
                               <FormattedMessage id="reports_partials_campaigns.last_click_date" />{' '}
-                              <span>{state.campaignSummaryResults.lastClickDate}</span>
+                              <span>
+                                <FormattedDate value={state.campaignSummaryResults.lastClickDate} />
```
Comment (andresmoschini) [inline src/components/Reports/ReportsPartialsCampaigns/ReportsPartialsCampaigns.js:201]: Dates are nice now, and other numbers?

image

I mean, are we not using thousand separators?

Comment (fcoronelmakingsense) [reply src/components/Reports/ReportsPartialsCampaigns/ReportsPartialsCampaigns.js:237]: changed

Comment (andresmoschini) [reply src/components/Reports/ReportsPartialsCampaigns/ReportsPartialsCampaigns.js:237]: Thanks, but then I realized that we are not formatting numbers in reports 🤦‍♂️, sorry.

Maybe we decided to not format them?

Well, decide it with the team, sorry by the noise.

**Pull request merged**